setup.py

```diff
@@ -379,6 +379,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
 ]
 sendgrid = [
     'sendgrid>=6.0.0,<7',
+    'starkbank-ecdsa>=1.0.0'  # PR 901 on sendgrid
```

Comment:
Can you do it as a separate PR? This will make it easier to cherry-pick this change to Airflow 1.10